kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -281,46 +289,45 @@ define([
                         }
                     }
                 }, {
-                    aTargets: [1],
-                    sClass: 'staging-name',
-                    mRender: function (data, type, full) {
+                    targets: 2,
+                    render: function (data, type, full) {
                         if (type === 'display') {
                             let decompressButton = '';
 
                             if (data.match(/\.(zip|tar\.gz|tgz|tar\.bz|tar\.bz2|tar|gz|bz2)$/)) {
-                                decompressButton = '<button class="btn btn-default btn-xs kb-data-staging-decompress" data-decompress="' + data + '"><i class="fa fa-expand"></i></button>';
+                                decompressButton = '<button class="btn btn-default btn-xs kb-staging-table-body__decompress" data-decompress="' + data + '"><i class="fa fa-expand"></i></button>';
                             }
 
                             if (full[0] === 'true') {
-                                data = '<span class="kb-data-staging-folder" data-name="' + data + '">' + data + '</span>';
+                                data = '<span class="kb-staging-table-body__folder" data-name="' + data + '">' + data + '</span>';
                             }
 
-                            return '<div class="kb-data-staging-table-name">' + decompressButton +
+                            return '<div class="kb-staging-table-body__name">' + decompressButton +
                                 data +
                                 '</div>';
                         }
                         return data;
                     }
                 }, {
-                    aTargets: [2],
-                    mRender: function (data, type) {
+                    targets: 3,
+                    type: 'num',
+                    render: function (data, type) {
                         if (type === 'display') {
                             return StringUtil.readableBytes(Number(data));
                         } else {
                             return Number(data);
                         }
-                    },
-                    sType: 'numeric'
+                    }
                 }, {
-                    aTargets: [3],
-                    mRender: function (data, type) {
+                    targets: 4,
+                    type: 'num',
+                    render: function (data, type) {
                         if (type === 'display') {
                             return TimeFormat.getShortTimeStampStr(Number(data));
                         } else {
                             return data;
                         }
-                    },
-                    sType: 'numeric'
+                    }
                 }],
                 rowCallback: function (nRow, aData, iDisplayIndex, iDisplayIndexFull) {
```

Comment:
It's not crucial, but it might be more user/programmer-friendly to rename `nRow` to `row`. The other three vars aren't used AFAICS, so they can be deleted.